src/ShopBundle/objects/db/TShopPaymentHandler/TShopPaymentHandlerPayPal_PayViaLink.class.php

```diff
@@ -177,7 +192,13 @@ public function ProcessIPNRequest($oOrder, $aURLParameter)
 
         if (false === $bIsComplete) {
             // some other response other than is paid - we log the info
-            $logger->error("PayPal IPN: returned payment status '{$aURLParameter['payment_status']}'! ".$aURLParameter.' data: '.$sData);
+            $logger->error(
+                'PayPal IPN: returned payment status not recognized.',
+                [
+                    'url_parameters' => $aURLParameter,
+                    'data' => $sData,
```

Comment:
please add order id and ordernumber to the context

Reply:
ok